cypress/e2e/tests/upgrade/create_upgrade_data.test.ts

```diff
@@ -110,6 +113,20 @@ describe(["@pre-upgrade"], "Creating pre-requisites before an upgrade", () => {
         tag.create();
     });
 
+    it("Creating and assess archetype", function () {
+        const { tagName, archetypeName } = this.upgradeData;
+        archetype = new Archetype(
+            archetypeName,
+            ["EJB XML", "Servlet"],
+            [tagName],
+            null,
+            [stakeHolder],
+            [stakeHolderGroup]
+        );
+        archetype.create();
+        archetype.perform_assessment("low");
```

Comment:
@ibragins , this is missing the verification of the completion of archetype assessment  . This could be committed through a follow up PR.
archetype.validateAssessmentField("Low");

Reply:
There is a bug on 7.0.2, assessment status doesn't change in UI regardless assessment was performed, this is why I didn't include status validation. Otherwize test will fail.